core/base/Module.php

```diff
@@ -178,6 +178,16 @@ public function getUrlRules()
      */
     public $moduleLayout = 'layout';
 
+    /**
+     * @var string The default file of layout
+     */
+    public $layout = 'main';
```

Comment:
layouts must be defined in the controller, i don't think we can introduce this variable here.